js/page-specific/baltcogo-app/constants/constants.js

```diff
@@ -0,0 +1,24 @@
+(function constantsWrapper(app) {
+	'use strict';
+
+	app.factory('CONSTANTS', constants);
```

Comment:
not that this is a huge deal, but probably should be written this way. I'm not sure of whether the constants object should be a variable or just passed in directly.

```javascript
app.constants('CONSTANTS', {
  urls: {}
});
```
or 

```javascript
app.constants('CONSTANTS', constants);

var constants = {
  urls: {}
};
```